modules/products/api/products-routes.ts

```diff
@@ -156,7 +157,7 @@ module.exports = function (app: core.Express) {
 	 *         schema:
 	 *           $ref: '#/definitions/MockProduct'
 	 */
-	app.get('/products/:qty?', (req: Request, res: Response) => {
+	app.get('/products/quantity/:qty?', (req: Request, res: Response) => {
```

Comment:
I do not feel we need /quantity here however I do like the use of /deparment above :)

Reply:
@ageddesi I have removed and PR should be updated.